dftimewolf/cli/recipes/activity_triage.py

```diff
@@ -0,0 +1,52 @@
+"""Collect artifacts from hosts using GRR.
+
+- Collect a predefined list of artifacts from hosts using GRR
+- Process them locally with grep to extract keywords
+"""
+from __future__ import unicode_literals
+
+_short_description = ('Fetches requested artifacts from a list of GRR hosts '
+                      'and processes them with grep.')
+
+contents = {
+    'name':
+        'activity_triage',
```

Comment:
activity_triage is even more generic ;) What's wrong with artifact_download_grep, or something similar? That's what in the description....

Reply:
Done. Changed to "artifact_grep"